app/models/event.rb

```diff
@@ -8,6 +8,7 @@ class Event < ApplicationRecord
     appropriate_body_releases_teacher
     appropriate_body_fails_teacher
     appropriate_body_passes_teacher
+    support_revert_teacher_claim
```

Comment:
I think this should be `admin_` as the class ~should~ will soon be callable from the admin console.